GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs

```diff
@@ -224,6 +224,14 @@ private IReadOnlyList<MenuCommand> CreateViewMenuCommands()
                     ExecuteAction = () => _revisionGrid.ToggleShowReflogReferences(),
                     IsCheckedFunc = () => AppSettings.ShowReflogReferences
                 },
+                new MenuCommand
+                {
+                    Name = "HideStashBranch",
+                    Text = "Hide stash branch",
```

Comment:
The text needs to be "Show..." to be consistent with the rest.
Somehow "stash branch" feels weird to me... Should it be "Show stash references"?

Reply:
https://github.com/gitextensions/gitextensions/pull/9555#pullrequestreview-745050308
and
https://github.com/gitextensions/gitextensions/pull/9555#discussion_r701074711

Reply:
I agree, it felt weird putting "hide" when all was "show". I is changed to the suggestion.